### PR TITLE
Add Version History

### PR DESCRIPTION
**Pull request by sohchatt**
Considered having separate version histories for Lottie-Windows, LottieViewer, LottieGen but concluded that it might too confusing. Open to suggestions about moving this content around / renaming the file.

### CODE REVIEW

File: Lottie-Windows/README.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
 # Lottie-Windows
-The Lottie-Windows project generates a NuGet package for use by Windows apps.
+Lottie-Windows produces the [`LottieVisualSource`](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.lottie.lottievisualsource) which is consumed by the [`Microsoft.UI.Xaml.Controls.AnimatedVisualPlayer`](https://docs.microsoft.com/uwp/api/microsoft.ui.xaml.controls.animatedvisualplayer) to render Lottie JSON files.
```
Comment (simeoncran) [inline Lottie-Windows/README.md]: Consider "provides".

File: Lottie-Windows/README.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
 ## Package locations
-* The latest release and pre-release versions are on NuGet: https://www.nuget.org/packages/Microsoft.Toolkit.Uwp.UI.Lottie.
-* The latest CI builds are on MyGet https://dotnet.myget.org/feed/uwpcommunitytoolkit/package/nuget/Microsoft.Toolkit.Uwp.UI.Lottie.
+* The [latest release and pre-release versions are on NuGet](https://www.nuget.org/packages/Microsoft.Toolkit.Uwp.UI.Lottie).
+* The [latest CI builds are on MyGet](https://dotnet.myget.org/feed/uwpcommunitytoolkit/package/nuget/Microsoft.Toolkit.Uwp.UI.Lottie).
 * The latest local build is output to the bin\nupkg directory in your repo directory.
+
+## Usage
+
+To get started with using the Lottie-Windows library in your XAML, follow [this tutorial](https://docs.microsoft.com/windows/communitytoolkit/animations/lottie-scenarios/getting_started_json).
```
Comment (simeoncran) [inline Lottie-Windows/README.md]: "with" is redundant and feels awkward to me.

File: LottieGen/README.md [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+LottieGen is a tool for generating C#, C++, and other outputs from Lottie / Bodymovin JSON files. LottieGen is built as a [.NET Core global tool](https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools), which means it will run anywhere .NET Core is available, including Linux and Mac.
 
-LottieGen is built as a [.NET Core global tool](https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools), which means it will run anywhere .NET Core is available, including Linux and Mac.
+LottieGen requires [.NET Core 2.2 SDK](https://dotnet.microsoft.com/download/dotnet-core/2.2) or later.
 
 ## Installing
 *The following commands are examples only; adjust the paths and versions as necessary.*
 
-The latest release version can be installed from NuGet (https://www.nuget.org/packages/LottieGen):
+The latest release version can be [installed from NuGet](https://www.nuget.org/packages/LottieGen):
```
Comment (simeoncran) [inline LottieGen/README.md]: Now would be a good time to update the version numbers to reflect the reality of 5.1

File: LottieGen/README.md [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
 
     dotnet tool install -g LottieGen
 
 The latest prerelease version can be installed from NuGet:
 
     dotnet tool install -g LottieGen --version 5.0.0-prerelease
```
Comment (simeoncran) [inline LottieGen/README.md]: There's also "dotnet tool update -g LottieGen". Easier that uninstalling and reinstalling.

File: README.md [hunk 5]
```diff
@@ -1,28 +1,36 @@
...
+Lottie-Windows is a library for rendering [Adobe AfterEffects](https://www.adobe.com/products/aftereffects.html) animations natively in your application. This project adds Windows to the [Lottie](http://airbnb.io/lottie/) family of tools also targeting [Android](https://github.com/airbnb/lottie-android), [iOS](https://github.com/airbnb/lottie-ios), and [Web](https://github.com/airbnb/lottie-web).
 
-Lottie-Windows works directly with JSON, but for even better performance it can codegen optimized C# and C++ code.
+Lottie simplifies the design-to-code workflow for bringing engaging, interactive vector animations to your Windows applications, with significant improvements in terms of performance, quality, and engineering efficiency over traditional approaches such as gifs, manually coded animations, etc. Lottie-Windows uses the [Windows.UI.Composition APIs](https://docs.microsoft.com/windows/uwp/composition/visual-layer) to provide smooth 60fps animations and resolution-independent vector graphics.
+
+![Lottie-Windows Gif](/images/LottieWindows_Intro.gif)
 
 Lottie-Winddows consists of 3 related products:
```
Comment (simeoncran) [inline README.md]: Typo: dd

File: VERSION_HISTORY.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
+# Version History
+
+Lottie-Windows aligns its major version with the Windows Community Toolkit.
+
+## 5.1.0 — Initial Release
+
+The following Adobe AfterEffects features are supported:
+
+**Layer types**
```
Comment (simeoncran) [inline VERSION_HISTORY.md]: I still don't understand what anyone will do with this list. Can't help thinking that a list of things that Lottie-Windows can't do would be more concise.

Comment (sohchatt) [reply VERSION_HISTORY.md]: Useful to understand the scope of what to design against. As we add more features, would be useful which versions support which.

**Pull request merged**